packages/store/addon/-private/store-service.ts

```diff
@@ -128,7 +128,7 @@ export interface CreateRecordProperties {
   }
   ```
 
-  You can learn more about writing a custom adapter by reading the `Adapter`
+  You can learn more about writing a custom adapter by reading the [Adapter](https://guides.emberjs.com/release/models/customizing-adapters/)
```

Comment:
thanks! probably the adapter docs would be better, the guides are very outdated:

- implementation details: https://api.emberjs.com/ember-data/release/classes/MinimumAdapterInterface
- overview: https://api.emberjs.com/ember-data/release/modules/@ember-data%2Fadapter

Reply:
hi thanks! @runspired  

I changed the link to the overview link that you shared.